ArduCopter/GCS_Mavlink.cpp

```diff
@@ -616,15 +616,26 @@ MAV_RESULT GCS_MAVLINK_Copter::handle_command_mount(const mavlink_command_long_t
 #if MOUNT == ENABLED
     case MAV_CMD_DO_MOUNT_CONTROL:
         if(!copter.camera_mount.has_pan_control() || (mount == nullptr)) {
-            copter.flightmode->auto_yaw.set_fixed_yaw(
-                (float)packet.param3 * 0.01f,
-                0.0f,
-                0,
-                0);
+//            copter.flightmode->auto_yaw.set_fixed_yaw(
```

Comment:
Remove commented code

Reply:
That block is there for easy A-B testing of the fault trigger; I'll remove once testing is complete. Basically, uncommenting the block fakes the conditions that lead to "snapping to north", which should be adequately handled by the rest of the code.